mne/time_frequency/psd.py

```diff
@@ -152,10 +350,17 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
     -------
     psds : ndarray (n_epochs, n_channels, n_freqs)
         The power spectral densities.
-    freqs : ndarray (n_freqs)
+    freqs : ndarray, shape (n_freqs)
```

Comment:
and it should actually be `shape (n_freqs,)`, since `(n_freqs)` is not a tuple but `(n_freqs,)` is (and `shape` is always a tuple)